pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -351,6 +351,28 @@ Answer.Picklist:
           type: array
           items:
             $ref: '../pepper.yml#/components/schemas/SelectedOption'
+Answer.File:
```

Comment:
Can you expand on what's going on here? It doesn't look like there's enough info for a `PATCH` to connect an answer to a file upload in either the `Answer.File` or `FileInfo` structures.

Reply:
The proposal is that the PATCH will have the same shape of a text or boolean question answer, but the `value` property will be the upload id that was. obtained in the same POST body that provided the upload URL. The screenshot below captures the intent:
<img width="741" alt="Screen Shot 2021-01-05 at 2 03 42 PM" src="https://user-images.githubusercontent.com/29984380/103687820-e60c0680-4f5e-11eb-8a0d-3d4704b0d5ca.png">

The chunk of YAML here is mean to describe the answer value that would appear when we GET the activity instance contents and we have a File answer.

Reply:
That's an awkward interaction. There shouldn't be 2 types of "Answer" objects.

Reply:
I recognize there is a bit of asymmetry between the PATCH body and the JSON we are including as the answer for File question in the activity instance.
Root cause is the difference between what the client needs to tell the server ("I just completed that upload you authorized me earlier to do") vs what the client needs to know to display something reasonable in the completed activity instance web page (file name and file size).  I think is fine. The file answer handling was always going to be a little different than the other answers.